community/modules/scheduler/schedmd-slurm-gcp-v6-controller/login.tf

```diff
@@ -59,26 +59,30 @@ module "slurm_login_template" {
 
 # INSTANCE
 module "slurm_login_instance" {
-  source   = "github.com/GoogleCloudPlatform/slurm-gcp.git//terraform/slurm_cluster/modules/slurm_login_instance?ref=6.4.3&depth=1"
+  source   = "github.com/GoogleCloudPlatform/slurm-gcp.git//terraform/slurm_cluster/modules/_slurm_instance?ref=trigger_happy&depth=1"
```

Comment:
You still have your branch reference in here.

Reply:
That's why it's labeled as "do-not-merge".